index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+
+    <link rel="stylesheet" href="./asset/css/style.css">
+</head>
+<body>
+    <header>
+        <div>
+            <span id="logo">My Logo</span>
```

Comment:
This content should be wrapped in an `<a>` tag. Please fix it.

<img width="442" alt="Screen Shot 2022-06-29 at 18 15 23" src="https://user-images.githubusercontent.com/82426682/176561197-56ab4e62-3ada-4f99-88ef-6340dd6eb499.png">

Reply:
Done